client-web/src/exlibris-app/shell/redux-store.html

```diff
@@ -0,0 +1,11 @@
+<link rel="import" href="../../../bower_components/polymer-redux/polymer-redux.html">
+
+<script>
+    const initialState = {};
+    const reducer = (state = initialState, action) => {
+    };
```

Comment:
`inititalState` and `reducer` will be also set to window. Use ES6 block scope: http://es6-features.org/#BlockScopedFunctions